internal/collector/processor/nodebatcher/metrics.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2019, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package nodebatcher
+
+import (
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+
+	"github.com/census-instrumentation/opencensus-service/internal/collector/processor"
+	"github.com/census-instrumentation/opencensus-service/internal/collector/telemetry"
+)
+
+var (
+	statBatchSize         = stats.Int64("batch_size", "Size of batches sent from the batcher (in span)", stats.UnitDimensionless)
+	statAddNodeBatches    = stats.Int64("add_node_batches", "Count of nodes that have been added to batching.", stats.UnitDimensionless)
+	statRemoveNodeBatches = stats.Int64("remove_node_batches", "Number of nodes that have been removed from batching.", stats.UnitDimensionless)
+
+	statSendByBatchSize     = stats.Int64("batch_size_trigger_send", "Number of times the batch was sent due to a size trigger", stats.UnitDimensionless)
```

Comment:
`statCountOfBatchingBySizeTrigger` -- `Number of times that batches were sent because of a size trigger`